js/web/lib/wasm/jsep/webgpu/gpu-data-manager.ts

```diff
@@ -157,6 +184,13 @@ class GpuDataManagerImpl implements GpuDataManager {
     this.buffersPending = [];
     this.externalBuffers = new Map();
     this.capturedPendingBuffers = new Map();
+
+    // eslint-disable-next-line guard-for-in
```

Comment:
`// eslint-disable-next-line guard-for-in` can be removed

Reply:
done